web/src/main/java/se/inera/intyg/webcert/web/service/facade/patient/GetPatientFacadeServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+package se.inera.intyg.webcert.web.service.facade.patient;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.common.support.facade.model.Patient;
+import se.inera.intyg.common.support.facade.model.PersonId;
+import se.inera.intyg.infra.integration.pu.model.PersonSvar;
+import se.inera.intyg.infra.integration.pu.services.PUService;
+import se.inera.intyg.schemas.contract.InvalidPersonNummerException;
+import se.inera.intyg.schemas.contract.Personnummer;
+import se.inera.intyg.webcert.web.service.monitoring.MonitoringLogService;
+import se.inera.intyg.webcert.web.web.controller.facade.PatientController;
+import se.inera.intyg.webcert.web.web.controller.facade.dto.PatientResponseDTO;
+
+@Service
+public class GetPatientFacadeServiceImpl implements GetPatientFacadeService {
+    private static final Logger LOG = LoggerFactory.getLogger(PatientController.class);
```

Comment:
Should the logger be instantiated with the current class?
```suggestion
    private static final Logger LOG = LoggerFactory.getLogger(GetPatientFacadeServiceImpl.class);
```